magetab_validation.py

```diff
@@ -20,25 +19,28 @@
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('idf',
-                        help="name of MAGE-TAB IDF file")
-
+                        help="Path to the MAGE-TAB IDF file")
+    parser.add_argument('-d', "--data_dir", default="",
+                        help="Path to the directory with SDRF and data files")
+    parser.add_argument('-v', '--verbose', action='store_const', const=10, default=20,
+                        help="Option to output detailed logging (debug level).")
     args = parser.parse_args()
 
-    return args.idf
+    return args.idf, args.data_dir, args.verbose
 
 
 def main():
     process_name = "magetab_validation"
 
-    idf_file = parse_args()
+    idf_file, data_dir, detail = parse_args()
     file_exists(idf_file)
 
     # Create logger
     current_dir, idf_file_name = os.path.split(idf_file)
-    logger = create_logger(current_dir, process_name, idf_file_name, logger_name="Validation")
+    logger = create_logger(current_dir, process_name, idf_file_name, logger_name="Validation", log_level=detail)
```

Comment:
You probably want to make log level configurable, as in production run you don't want to leave this in detailed mode.

Reply:
Maybe the choice of variable name isn't great here. "detail" stores the logging level, default is 20 (which is INFO) and the -v option sets it to 10 (which is DEBUG). I'll rename to "logging_level".